frontend/src/views/ManagerLanding.vue

```diff
@@ -0,0 +1,144 @@
+<template>
+  <div>
+    <v-card elevation="0" class="text-center text-md-left">
+      <v-row v-if="$vuetify.breakpoint.smAndDown">
+        <v-col cols="12">
+          <v-img max-height="150px" contain src="/static/images/doodles/primary/SittingChef.png"></v-img>
+        </v-col>
+      </v-row>
+      <v-row>
+        <v-col cols="12" md="9">
+          <v-card-title>
+            <v-spacer></v-spacer>
+            <h1 class="font-weight-black text-h3 mt-0 mt-sm-4">
```

Comment:
Je te propose qu'on change le _heading_ en `xs`, sinon ce pavé de texte prend beaucoup trop de place. Par ex, la capture d'écran ci-dessous sur in iPhone 11.

On peut faire par exemple `class="... text-h5 text-sm-h4 text-md-h3 ..."`, ça rendrait mieux je pense.

![image](https://user-images.githubusercontent.com/1225929/167129260-7725f6a9-f542-401d-8089-a34ba7e8c160.png)

Reply:
Nickel